examples/logging/src/main/java/io/opentelemetry/example/logging/LoggingExporterExample.java

```diff
@@ -16,8 +16,8 @@
   private final Tracer tracer;
   private final LongCounter counter;
 
-  public LoggingExporterExample(OpenTelemetry openTelemetry) {
-    tracer = openTelemetry.getTracer(INSTRUMENTATION_NAME);
+  public LoggingExporterExample() {
+    tracer = GlobalOpenTelemetry.getTracer(INSTRUMENTATION_NAME);
```

Comment:
Ah sorry for the confusion, we still use the returned instance here. The global is only to support special instrumentation such as mysql.

Reply:
gotcha sorry will revert